erpnext/accounts/doctype/asset/asset.py

```diff
@@ -1,260 +1,10 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
+# Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
 import frappe
-from frappe import _
-from frappe.utils import flt, add_months, cint, nowdate, getdate
 from frappe.model.document import Document
-from erpnext.accounts.doctype.purchase_invoice.purchase_invoice import get_fixed_asset_account
-from erpnext.accounts.doctype.asset.depreciation \
-	import get_disposal_account_and_cost_center, get_depreciation_accounts
 
 class Asset(Document):
```

Comment:
Same as previous comment. If this is deleted .. delete the doctype not just the code.